aten/src/ATen/cpu/vec/vec_mask.h

```diff
@@ -134,6 +134,14 @@ class VecMask {
     return VectorizedN<T, N>(VectorizedN<T, N>::loadu(mask));
   }
 
+  void store(bool* b, int count) {
```

Comment:
```suggestion
  void store_mask(bool* b, int count) {
```

Reply:
There are several places in cpp.py using `store` instead of `store_mask`, such as:

- https://github.com/pytorch/pytorch/blob/ec00daf4f1b09b830a2dfc300a859b5d6f9f39e5/torch/_inductor/codegen/cpp.py#L2449-L2453
- https://github.com/pytorch/pytorch/blob/ec00daf4f1b09b830a2dfc300a859b5d6f9f39e5/torch/_inductor/codegen/cpp.py#L2343
- https://github.com/pytorch/pytorch/blob/ec00daf4f1b09b830a2dfc300a859b5d6f9f39e5/torch/_inductor/codegen/cpp.py#L2815

I afraid we need to check the dtype in all the places and change `store` to `store_mask`?

Reply:
oh, that's good point. I was just concerned that the name might be confusing with "masked store" semantics. Perhaps, we'd better rename `storeu` with `masked_storeu`..